### DISCUSSION TITLE
I ported over svelte's spring motion library into an alpine plugin

### DISCUSSION THREAD

**Discussion by nasheomirro** [alpinejs/alpine#4119] [2. Show and tell]
I wanted a quick way to animate values in a springy motion and I really liked svelte's `spring` store for doing exactly that, I missed it so much I went and ~stole~ ported it over to Alpine. You can check the [git repo here](https://github.com/nasheomirro/alpinejs-spring), and to quickly [try it out here](https://codepen.io/shanuthrii/pen/MWROjqW).

**Comment by ekwoka** [comment]
Thats pretty cool!!!
